test/integration/scene-stream-access/get-scene-stream-access-handler.spec.ts

```diff
@@ -0,0 +1,253 @@
+    identity: string
+    realmName: string
+    parcel: string
+    hostname: string
+    sceneId: string
+  }
+
+  let metadataLand: Metadata
+  let metadataWorld: Metadata
+  let mockIngress: IngressInfo
+  let mockSceneStreamAccess: any
+
+  beforeEach(async () => {
+    cleanup = new TestCleanup(components.database)
+
+    mockIngress = {
+      id: 'mock-ingress-id',
+      name: 'mock-ingress',
+      url: 'rtmp://mock-stream-url',
+      streamKey: 'mock-stream-key',
+      ingressId: 'mock-ingress-id'
+    } as IngressInfo
+
+    mockSceneStreamAccess = {
+      id: 'mock-access-id',
+      place_id: placeId,
+      streaming_url: 'rtmp://mock-stream-url',
+      streaming_key: 'mock-stream-key',
+      ingress_id: 'mock-ingress-id',
+      created_at: Date.now(),
+      active: true
+    }
+
+    metadataLand = {
+      identity: owner.authChain[0].payload,
+      realmName: 'test-realm',
+      parcel: '10,20',
+      hostname: 'https://peer.decentraland.zone',
+      sceneId: 'test-scene'
+    }
+
+    metadataWorld = {
+      identity: owner.authChain[0].payload,
+      realmName: 'name.dcl.eth',
+      parcel: '20,20',
+      hostname: 'https://worlds-content-server.decentraland.org/',
+      sceneId: 'test-scene'
+    }
+
+    jest.spyOn(handlersUtils, 'validate').mockResolvedValue(metadataLand)
+    jest.spyOn(components.sceneFetcher, 'getPlace').mockResolvedValue({
+      id: placeId,
+      positions: ['10,20'],
+      owner: owner.authChain[0].payload
+    } as PlaceAttributes)
+    jest.spyOn(components.sceneFetcher, 'hasLandPermission').mockResolvedValue(true)
+    jest.spyOn(components.sceneFetcher, 'hasWorldPermission').mockResolvedValue(false)
+    jest.spyOn(components.sceneAdminManager, 'isAdmin').mockResolvedValue(false)
+    jest.spyOn(components.livekit, 'getSceneRoomName').mockReturnValue(`test-realm:test-scene`)
+    jest.spyOn(components.livekit, 'getWorldRoomName').mockReturnValue(`name.dcl.eth`)
+    jest.spyOn(components.livekit, 'getOrCreateIngress').mockResolvedValue(mockIngress)
+    jest.spyOn(components.sceneStreamAccessManager, 'getAccess').mockResolvedValue(mockSceneStreamAccess)
+    jest.spyOn(components.sceneStreamAccessManager, 'addAccess').mockResolvedValue(undefined)
+  })
+
+  afterEach(async () => {
+    await cleanup.cleanup()
+    jest.restoreAllMocks()
+  })
+
+  it('returns 200 with streaming access when user has land permission', async () => {
+    const { localFetch } = components
+
+    const response = await makeRequest(
+      localFetch,
+      '/scene-stream-access',
+      {
+        method: 'GET',
+        metadata: metadataLand
+      },
+      owner
+    )
+
+    expect(response.status).toBe(200)
+    const body = await response.json()
+    expect(body).toHaveProperty('streaming_url')
+    expect(body).toHaveProperty('streaming_key')
+    expect(body.streaming_url).toBe(mockSceneStreamAccess.streaming_url)
+    expect(body.streaming_key).toBe(mockSceneStreamAccess.streaming_key)
```

Comment:
If you want, this can be simplified in a single line:
```suggestion
    expect(response.json).resolves.toEqual({
      "streaming_url": mockSceneStreamAccess.streaming_url,
      "streaming_key": mockSceneStreamAccess.streaming_key
    })
```